### ISSUE TITLE
Missing Authorize on some API's endpoints

### ISSUE THREAD

**Issue by JonanOribe** [LAION-AI/Open-Assistant#2805]
Looking at the API, neither this endpoint

![image](https://user-images.githubusercontent.com/11914201/233580790-61853ff7-a46d-4116-9038-c375720602ae.png)

or this one 

![image](https://user-images.githubusercontent.com/11914201/233580885-16f6d381-4d0e-49df-a000-93d19a36b338.png)

Ask for credentials. There is a reason for that? if not, I can update those

**Comment by yk** [comment]
there is not really any reason, the data there is not confidential per se, but yes it makes sense to have auth there. if you're up to it, happy to take a PR :)

**Comment by andreaskoepf** [comment]
@JonanOribe Are you actively working on a PR for the endpoint auth?

**Comment by JonanOribe** [comment]
Hi @andreaskoepf, I already solve the issue with the "report labels" endopint at [https://github.com/LAION-AI/Open-Assistant/pull/3042](url) . What I can't find is how to modify the "/metrics" endpoint. It looks that comes directly from prometheus instrumentator.

![image](https://user-images.githubusercontent.com/11914201/236484360-84cdee78-d5a5-4014-bf9a-39f2c614dfc9.png)

I don't know if you could guide me a bit.Thanks!

**Comment by andreaskoepf** [comment]
> It looks that comes directly from prometheus instrumentator.

Yes, maybe [this issue](https://github.com/trallnag/prometheus-fastapi-instrumentator/issues/132) could be a starting point? 
BTW Currently prometheus relies on unauthenticated metrics endpoints .. If you send a PR for this we need to update the prometheus configuration.

**Comment by JonanOribe** [comment]
Thanks, I will take a look around that. Meanwhile, I leave  [here](https://github.com/LAION-AI/Open-Assistant/pull/3043) the PR for the other endpoint.
